src/main/java/github/scarsz/discordsrv/api/commands/PluginSlashCommand.java

```diff
@@ -137,4 +154,57 @@ public PluginSlashCommand setPriority(SlashCommandPriority priority) {
         return this;
     }
 
+    public Plugin getPlugin() {
+        return this.plugin;
+    }
+
+    public CommandData getCommandData() {
+        return this.commandData;
+    }
+
+    public Set<String> getGuilds() {
+        return this.guilds;
+    }
+
+    public SlashCommandPriority getPriority() {
+        return this.priority;
+    }
+
+    public boolean equals(final Object o) {
+        if (o == this) return true;
+        if (!(o instanceof PluginSlashCommand)) return false;
+        final PluginSlashCommand other = (PluginSlashCommand) o;
+        final Object this$plugin = this.getPlugin();
+        final Object other$plugin = other.getPlugin();
+        if (this$plugin == null ? other$plugin != null : !this$plugin.equals(other$plugin)) return false;
+        final Object this$commandData = this.getCommandData();
+        final Object other$commandData = other.getCommandData();
+        if (this$commandData == null ? other$commandData != null : !this$commandData.equals(other$commandData))
+            return false;
+        final Object this$guilds = this.getGuilds();
+        final Object other$guilds = other.getGuilds();
+        if (this$guilds == null ? other$guilds != null : !this$guilds.equals(other$guilds)) return false;
+        final Object this$priority = this.getPriority();
+        final Object other$priority = other.getPriority();
+        if (this$priority == null ? other$priority != null : !this$priority.equals(other$priority)) return false;
+        return true;
+    }
+
+    public int hashCode() {
+        final int PRIME = 59;
+        int result = 1;
+        final Object $plugin = this.getPlugin();
+        result = result * PRIME + ($plugin == null ? 43 : $plugin.hashCode());
+        final Object $commandData = this.getCommandData();
+        result = result * PRIME + ($commandData == null ? 43 : $commandData.hashCode());
+        final Object $guilds = this.getGuilds();
+        result = result * PRIME + ($guilds == null ? 43 : $guilds.hashCode());
+        final Object $priority = this.getPriority();
+        result = result * PRIME + ($priority == null ? 43 : $priority.hashCode());
+        return result;
+    }
```

Comment:
This should use [Objects.hash](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#hash-java.lang.Object...-) instead